src/Api/FingerprintApi.php

```diff
@@ -128,29 +128,17 @@ public function getEvent(string $request_id): array
 
             switch ($e->getCode()) {
                 case 200:
-                    $data = ObjectSerializer::deserialize(
-                        $response,
-                        '\Fingerprint\ServerAPI\Model\EventResponse'
-                    );
-                    $e->setResponseObject($data);
+                    $e->setResponseObject($response);
 
                     break;
 
                 case 403:
-                    $data = ObjectSerializer::deserialize(
-                        $response,
-                        '\Fingerprint\ServerAPI\Model\ErrorEvent403Response'
-                    );
-                    $e->setResponseObject($data);
+                    $e->setResponseObject($response);
 
                     break;
 
                 case 404:
-                    $data = ObjectSerializer::deserialize(
-                        $response,
-                        '\Fingerprint\ServerAPI\Model\ErrorEvent404Response'
-                    );
-                    $e->setResponseObject($data);
+                    $e->setResponseObject($response);
 
                     break;
             }
```

Comment:
Let's keep the deserialized response in the error, but in a different property, such as `$responseBody`. 
Right now these switch statements are redundant 😆